readme.md

```diff
@@ -591,6 +591,7 @@
 - [Open Source Maintainers](https://github.com/nayafia/awesome-maintainers#readme) - The experience of being an open source maintainer.
 - [Calculators](https://github.com/xxczaki/awesome-calculators#readme) - Calculators for every platform.
 - [Captcha](https://github.com/ZYSzys/awesome-captcha#readme) - A type of challenge–response test used in computing to determine whether or not the user is human.
+- [French](https://github.com/jusdepatate/awesome-french#readme) - An awesome list of french things.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.